pkg/operator/controller/prometheus.go

```diff
@@ -88,6 +79,10 @@ func ensurePrometheusResourcesExist(ctx context.Context, c client.Client, scheme
 			return err
 		}
 
+		switch rule := desired.(type) {
+		case *promv1.PrometheusRule:
+			log.Info("[DEBUG]", "desired prometheus object spec", rule.Spec)
+		}
```

Comment:
leftover?

Reply:
wow I don't know where is came from, removing.